test/nbrowser/ProposedChangesPage.ts

```diff
@@ -6,22 +6,79 @@ describe('ProposedChangesPage', function() {
   this.timeout(60000);
   const cleanup = setupTestSuite();
 
-  it('show comparison and functions as expected', async function() {
+  // Currently this page exists only on a document where accepting
+  // proposals is turned on. The flag for turning this on is, in turn, hidden
+  // behind an experimental flag.
+  it('can be enabled experimentally for a document', async function() {
     const session = await gu.session().teamSite.login();
-    // const api = session.createHomeApi();
     await session.tempDoc(cleanup, 'Hello.grist');
-    // Open the share menu and click item to work on a copy.
+    await driver.find('.test-tools-settings').click();
+
+    // Add the experimental flag to the URL.
+    const url = await driver.getCurrentUrl();
+    await driver.get(url + '?experiment=proposedChangesPage');
+    // Confirm the flag.
+    await driver.findWait('.test-modal-confirm', 2000).click();
+    // Confirm we want to reload.
+    await driver.findWait('.test-modal-confirm', 2000).click();
+
+    // Check the accepting proposals checkbox is now visible.
+    assert.match(
+      await driver.findWait('#admin-panel-item-description-acceptProposals', 2000).getText(),
+      /Allow others to propose changes/);
+    // But it shouldn't be checked yet.
+    assert.equal(
+      await driver.find('input.test-settings-accept-proposals').getAttribute('checked'),
+      null
+    );
+    // Now check it.
+    await driver.find('input.test-settings-accept-proposals').click();
+    // A new page should appear in the toolbox.
+    await driver.findWait('.test-tools-proposals', 2000);
+    // The flag should be checked now.
+    assert.equal(
+      await driver.find('input.test-settings-accept-proposals').getAttribute('checked'),
+      'true'
+    );
+  });
+
+  it('can make and apply a simple proposed change', async function() {
+    // Load a test document.
+    const session = await gu.session().teamSite.login();
+    const doc = await session.tempDoc(cleanup, 'Hello.grist');
+
+    // Turn on feature.
+    const api = session.createHomeApi();
+    await api.updateDoc(doc.id, {
+      options: {
+        proposedChanges: {
+          acceptProposals: true
+        }
+      }
+    });
+
+    // Put something known in the first cell.
     await gu.getCell('A', 1).click();
     await gu.waitAppFocus();
     await gu.enterCell('test1');
+
+    // Work on a copy.
     await driver.find('.test-tb-share').click();
-    await driver.find('.test-work-on-copy').click();
+    await driver.findWait('.test-work-on-copy', 2000).click();
     await gu.waitForServer();
+
+    // Change the content of the first cell.
     await gu.getCell('A', 1).click();
```

Comment:
You need to wait for the document to reload, or even, reload it manually to make sure it is reloaded properly.
I get `stale element` error in this line.